SF iOS/SF iOS/Feed/FeedItem.m

```diff
@@ -24,7 +25,7 @@ - (instancetype)initWithEvent:(Event *)event {
         return nil;
     }
     self.dateString = [self dateStringFromDate:event.date];
-    self.title = event.name;
+    self.title = event.venue.name;
```

Comment:
This is noteworthy. I'm Essentially dropping the event title in favor of the venue title which is shown on the detail view.

Reply:
Based on @gravicle's original implementation, this was the intended experience.

Reply:
Ok and we're back. Ignore the above comments
<img width="475" alt="Screen Shot 2019-04-16 at 12 22 53 PM" src="https://user-images.githubusercontent.com/75346/56238098-79dac800-6042-11e9-8112-1d780fcb1c2a.png">